init.sh

```diff
@@ -355,7 +355,7 @@ fi
 
 echo "## Restarting containers... "
 docker-compose down
-docker-compose -f docker-compose.yml -f "platform.$(uname -s).yml" up -d
+docker-compose -f docker-compose.yml -f "platform.$(uname -s).yml" up -d nginx
```

Comment:
Does this only restart nginx when we run `./init.sh`?

Reply:
`docker-compose down` will stop every running container that is defined in the docker-compose file.
`docker-compose up nginx` will only run ngnix, but as everything else is defined as a dependency, then every image (except for POW) will be run.

Reply:
Awesome. Great thinking